Correction d'une faute d'orthographe

Lucas-ech opened the pull request:
Une faute d'orthographe s'est glissée dans le message de confirmation lorsqu'on a changé son mot de passe via la fonction "Mot de passe oublié".

Lucas-ech committed:
Correction d'une faute d'orthographe

Une faute d'orthographe s'est glissée dans le message de confirmation lorsqu'on a changé son mot de passe via la fonction "Mot de passe oublié".

WinXaito commented:
Je ne suis pas sur mais il me semble que "Vous connectez" est juste, mais ça me parait vraiment bizarre à l'oeil.

J'aurais dit qu'en partant de "Connectez-vous", on garde le "ez"

Après personnellement j'aurais mieux vu comme messgae "Se connecter". Ca me paraît plus propre.

amorison commented:
Non, "Vous connecter" est la bonne forme (grammaticalement, c'est
exactement pareil que "Se connecter", c'est le "vous" en tant que pronom
objet de l'action, comme dans "nous vous aimons bien", pas en tant que
pronom "normal").

2015-08-26 23:05 GMT+02:00 WinXaito notifications@github.com:

> Je ne suis pas sur mais il me semble que "Vous connectez" est juste, mais
> ça me parait vraiment bizarre à l'oeil.
> 
> J'aurais dit qu'en partant de "Connectez-vous", on garde le "ez"
> 
> Après personnellement j'aurais mieux vu comme messgae "Se connecter". Ca
> me paraît plus propre.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/zestedesavoir/zds-site/pull/2968#issuecomment-135170125
> .

Situphen commented:
Mettre seulement "vous connecter" est correcte mais AMHA, ce serait mieux de mettre carrément "Votre mot de passe a bien été modifié, vous pouvez dès maintenant vous connecter" :

``` html+django
{% trans "Votre mot de passe a bien été modifié, vous pouvez dès maintenant" %} <a href="{% url "zds.member.views.login_view" %}">{% trans "vous connecter" %}</a>.
```

Par ailleurs, il y a une une autre faute dans ce fichier (elle est présente deux fois) : "La modification du mot de passe a été éffectuée". Ce n'est pas "éffectuée" mais "effectuée". @Lucas-ech, tu peux corriger celle-ci aussi ? :)

Commit:
Correction d'une faute d'orthographe

Correction des deux occurrences "éffectuée" et amélioration de la phrase
 de confirmation par @Situphen

WinXaito commented:
**QA** Ok pour moi (J'ai surtout regardé fonctionnalité, je n'ai pas fais attention s'il y avait d'autre faute mais je n'ai rien vu).

pierre-24 commented:
Vu :)
